### PR TITLE
Improve Security Policy

### PR DESCRIPTION
**Pull request by aepfli**
Adding a more detailed default process for our security vulnerabilities.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -1,6 +1,38 @@
...
+
+### When Should I NOT Report a Vulnerability?
+
+- You need help tuning one of our our Open Source projects for security - please discuss this with the maintainers of said project
+- You need help applying security-related updates
+- Your issue is not security-related
+
+## Security Vulnerability Response
+
+Each report is acknowledged and analyzed by the security contacts within 5 working days. This will set off the [Security Release Process](#process).
```
Comment (DavidPHirsch) [inline SECURITY.md]: would put this happens on a best effort

File: SECURITY.md [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
 
-## Reporting a Vulnerability
+This page borrows parts of its contents from https://kubernetes.io/security/
 
-Please report vulnerabilities to <opensource@dynatrace.com>.
+## Report a Vulnerability
+
+We are extremely grateful for security researchers and users that report vulnerabilities to the Dynatrace Open Source Community. All reports are thoroughly investigated by a set of community members.
+
+To make a report, submit your vulnerability to <opensource@dynatrace.com>. This allows triage and handling of the vulnerability with standardized response times.
```
Comment (DavidPHirsch) [inline SECURITY.md]: rather than standardized response time, this is best effort

File: SECURITY.md [hunk 3]
```diff
@@ -1,6 +1,38 @@
...
+
+## Public Disclosure Timing
+
+A public disclosure date is negotiated by the Dynatrace Open Source Community and the bug submitter. We prefer to fully disclose the bug as soon as possible once a user mitigation is available. It is reasonable to delay disclosure when the bug or the fix is not yet fully understood, the solution is not well-tested, or for vendor coordination. The timeframe for disclosure is from immediate (especially if it is already publicly known) to a few weeks. For a vulnerability with a straightforward mitigation, we expect report date to disclosure date to be on the order of 7 days. The Dynatrace Open Source Community holds the final say when setting a disclosure date.
+
+## Process
+
+If you find a security-related bug in a Dynatrace Open Source project, we kindly ask you for responsible disclosure and for giving us appropriate time to react, analyze, and develop a fix to mitigate the found security vulnerability. The security contact will investigate the issue within 5 working days.
```
Comment (DavidPHirsch) [inline SECURITY.md]: again would avoid any reference to days, it is all best effort for now

**Review by DavidPHirsch — CHANGES_REQUESTED** [review]
I left a few notes where it would be best to remove timeframes but rather have it as best effort

File: SECURITY.md (hunk 2 above)
Comment (aepfli) [reply SECURITY.md]: would you say `appropriate timeframe` would fit your needs?

Comment (DavidPHirsch) [reply SECURITY.md]: yes

**Review by DavidPHirsch — APPROVED** [review]

**Pull request merged**